src/pages/components/tile/style.mdx

```diff
@@ -28,99 +28,170 @@ and structure.
 
 ## Color
 
-<InlineNotification>
+### Base tile color
 
-The tile border applies to clickable, selectable and expandable tiles — not to
-base tiles.
+| Element   | Property         | Color token     |
+| --------- | ---------------- | --------------- |
+| Container | background-color | `$layer` \*     |
+| Text      | text color       | `$text-primary` |
 
-</InlineNotification>
+<Row>
+<Column colLg={8}>
+
+![Base tile color](images/tile-style-color-base.png)
+
+<Caption>Base tile color</Caption>
+
+</Column>
+</Row>
+
+### Clickable tile color
 
-| Element         | Property         | Color token         |
-| --------------- | ---------------- | ------------------- |
-| Tile:background | background-color | `$layer` \*         |
-| Tile border     | border           | `$border-tile` \*   |
-| Icon            | svg              | `$icon-interactive` |
-| Checkbox icon   | svg              | `$icon-primary`     |
-| Checkmark icon  | svg              | `$icon-primary`     |
-| Chevron icon    | svg              | `$icon-primary`     |
+Clickable tiles have an available
+[feature flag](https://carbondesignsystem.com/components/overview/feature-flag/).
+
+| Element   | Property         | Color token         |
+| --------- | ---------------- | ------------------- |
+| Container | background-color | `$layer` \*         |
+| Text      | text color       | `$text-primary`     |
+| Border    | border           | `$border-tile` \*   |
```

Comment:
I'm wondering if we should call out the feature flag elements. What do y'all think? Or if it obvious enough which elements before to which variant. 
```suggestion
| Border  (feature flag)  | border           | `$border-tile` \*   |
```